.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Will be cool exclude this file from the pull request

Reply:
Cool